### PR TITLE
Add quantized model export description.

### PR DESCRIPTION
**Pull request by linbinskn**
Add and example of saving and loading quantized model in QAT.

### CODE REVIEW

File: docs/en_US/Compression/QuickStart.md [hunk 1]
```diff
@@ -193,4 +195,18 @@ pruner.export_model(model_path='model.pth')
 pruner.export_model(model_path='model.pth', mask_path='mask.pth', onnx_path='model.onnx', input_shape=[1, 1, 28, 28])
 ```
 
+#### Export Quantized Model
+
+You can export the quantized model directly by using ```torch.save``` api and the quantized model can be loaded by ```torch.load``` without any extra modification. The following example shows the normal proceduce of saving and loading quantized model in QAT.
```
Comment (QuanluZhang) [inline docs/en_US/Compression/QuickStart.md:200]: better to briefly explain where is quantization related data saved, for example, scale, zero point etc.

File: docs/en_US/Compression/QuickStart.md [hunk 2]
```diff
@@ -193,4 +195,24 @@ pruner.export_model(model_path='model.pth')
 pruner.export_model(model_path='model.pth', mask_path='mask.pth', onnx_path='model.onnx', input_shape=[1, 1, 28, 28])
 ```
 
+#### Export Quantized Model
+
+You can export the quantized model directly by using ```torch.save``` api and the quantized model can be loaded by ```torch.load``` without any extra modification. The following example shows the normal proceduce of saving, loading quantized model and get related parameters in QAT.
+```
+# save quantized model
+torch.save(model.state_dict(), "quantized_model.pkt")
```
Comment (QuanluZhang) [inline docs/en_US/Compression/QuickStart.md:203]: it is not clear what the relation between `model` and `model_1` below...

Comment (linbinskn) [reply docs/en_US/Compression/QuickStart.md:203]: Have changed model_1 to qmodel_load and add related annotation.

File: docs/en_US/Compression/QuickStart.md (hunk 1 above)
Comment (linbinskn) [reply docs/en_US/Compression/QuickStart.md]: Agree.
